CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * [CHANGE] Querier: Remove `-querier.at-modifier-enabled` flag. #6157
 * [CHANGE] Tracing: Remove deprecated `oltp_endpoint` config entirely. #6158
 * [CHANGE] Store Gateway: Enable store gateway zone stable shuffle sharding by default. #6161
+* [FEATURE] Ruler: Minimize rule group missed evaluations via `-ruler.enable-ha` flag. #6129
```

Comment:
Need to rebase master and move this changelog entry to `master / unreleased` section

Reply:
Can we make the changelog more descriptive? As an end user, I don't understand what does it mean by `Minimize rule group missed evaluations`. This flag helps end users under which scenario? I see we added other new flags as well.

Reply:
And the flag name is `-ruler.enable-ha-evaluation`